buf/registry/module/v1beta1/label_service.proto

```diff
@@ -42,6 +42,8 @@ service LabelService {
   //
   // If the Label does not exist, it will be created.
   // If the Label was archived, it will be unarchived.
```

Comment:
wait, it will only be unarchived if a new commit is pushed, right?

Reply:
Good question. If label is `[A, B, C]` and archived, and I send a request to update it to `D`, I'd expect it to become `[A, B, C, D]` _and_ unarchived.

Reply:
agree, but if you try to update the label to point to a commit that is older than C, it should fail (which means that it should not become unarchived). The current docs here are written in a way that make it seem like it will always become unarchived.

Reply:
Later on down it says
> This operation is atomic. Either all Labels are created/updated or an error is returned.

So I read this as "if the request fails, no state is changed".

Reply:
ok, as long as we are all on the same page, that is what I care about. good enough to merge